.github/ISSUE_TEMPLATE/security-vulnerability.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Security Vulnerability
+about: Report vulnerability in the project
+---
+
+<!-- Thank you for your contribution. Before you submit the issue:
+1. Search open and closed issues for duplicates.
+2. Read the contributing guidelines.
+3. In case of vulnerabilities with a CVSS Base Score of *7.0* or above (see risk assessment section), please send an email directly to security@kyma-project.io instead of using this issue tracker. You can still use this template in your email. You can also use email for vulnerabilities with lower severity, if you prefer to keep your report confidential.
+-->
+
+**Description**
+
+<!-- Provide a clear and concise description of the vulnerability.
+Describe where it appears, when it occurred, and what it affects.  -->
+
+<!-- Provide relevant technical details such as the browser name and version, or the operating system. -->
+
+**Steps to exploit**
+
+<!-- List the steps an attacker would follow to exploit the vulnerability. Attach any files, links, code samples, or screenshots that could help in reproducing the attack. -->
+
+**Risk assessment**
+
+<!-- Propose a CVSSv3.0 Base Score for the vulnerability.
+Please use the CVSS calculator at https://www.first.org/cvss/calculator/3.0 and fill in the risk metrics for the CVSS Base Score. Then replace the placeholders in the following template: -->
+[{CVSS Base Vector}]({CVSS Calculator URL}) **{CVSS Base Score} ({CVSS Base Severity})**
```

Comment:
I'm aware that this template is somewhat bloated, and a little error prone. For example, copy-and-paste errors could lead to inconsistencies between vector, url, score, and severity.

However the proposed format displays nicely, and makes all necessary information readily available. Do you think we can keep this, or should we simplify?

Reply:
LGTM for now